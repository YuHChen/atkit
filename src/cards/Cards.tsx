import React from "react";

import Wip from "../Wip";

const Cards = () => <Wip pageName="cards page" />;

export default Cards;
