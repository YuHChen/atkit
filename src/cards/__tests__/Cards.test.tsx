import React from "react";
import { render } from "@testing-library/react";

import Cards from "../Cards";

describe("Cards", () => {
  test("renders as expected", () => {
    const { container } = render(<Cards />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
