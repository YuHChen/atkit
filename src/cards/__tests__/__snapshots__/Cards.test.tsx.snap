// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards renders as expected 1`] = `
<div
  class="fade mt-3 alert alert-info show"
  role="alert"
>
  <div
    class="alert-heading h4"
  >
    Pardon the mess
  </div>
  <p
    class="mb-0"
  >
    This 
    cards page
     is still a work in progress.
  </p>
</div>
`;
